### PR TITLE
Added management definitions

### PR DESCRIPTION
**Pull request by eduardo-rodrigues**
Many thanks to Chris Parkes for the input!

### CODE REVIEW

File: glossary/l.md [hunk 1]
```diff
@@ -19,10 +19,37 @@ See our [private](http://lhcb.web.cern.ch/lhcb/) or [public](http://lhcb-public.
...
+The committee is composed of senior physicists who are not heavily involved in LHC experiments
+or at least not in the experiments that they are reviewing.
+
 ## LFN: Logical File Name {#LFN}
 
 Logical File Name. A path for a file stored on the grid that does not depend on a file's physical location(s).
 
+## LoI: Letter of Intent {#LoI}
+
+An initial document submitted to the LHCC of CERN describing interest in performing a new experiment or major upgrade,
```
Comment (alexpearce) [inline glossary/l.md]: Link LHCC.

File: glossary/m.md [hunk 2]
```diff
@@ -4,6 +4,14 @@
 
 The LHCb trigger emulation application. [Project website](http://lhcbdoc.web.cern.ch/lhcbdoc/moore/) .
 
+## MoU: Memorandum of Understanding {#MoU}
+
+A document for the [RRB](r.md#RRB) stating the cost of construction of an experiment, or its major upgrade, or its maintenance and operations.
+The construction documents are broken down by subdetector and subsystem, and state the sum that will be paid by each
+of the funding agencies of the contributing countries. The document is signed by representatives of the funding agency.
+The costs given are those of all materials and does not include the cost of the effort in the contributing institutes.
```
Comment (alexpearce) [inline glossary/m.md]: Could we be clearer on 'effort'? I guess it means 'labour costs', so maybe '...does not include salary costs paid by the contributing institutes.'.

File: glossary/e.md [hunk 3]
```diff
@@ -22,6 +22,18 @@ This includes all devices in the areas of data acquisition, detector control (e.
 and the interaction with the outside world.
 Details in the Online System [webpages](http://lhcb-online.web.cern.ch/lhcb-online/ecs/default.htm).
 
+## EDR: Engineering Design Review {#EDR}
+
+A milestone during the construction of a subdetector.
+The EDR is a meeting with accompanying documentation that describes the detailed design of a component system of the detector
+(e.g. sensors, an ASIC, high voltage system) based on drawings and prototypes that have been produced.
```
Comment (alexpearce) [inline glossary/e.md]: I don't think we have an entry for ASIC. Seeing as it's not LHCb-specific, a link to the [Wikipedia article](https://en.wikipedia.org/wiki/Application-specific_integrated_circuit) could be enough.

File: glossary/r.md [hunk 4]
```diff
@@ -10,3 +10,10 @@ Gas-filled detectors that employ spherical mirrors to focus Cherenkov light into
 The radius of each ring is a function of the velocity of the particle, which, when combined with momentum (from the tracking system), provides discrimination between particles of different mass.
 In particular, the LHCb RICH discriminates between charged pions, charged kaons and protons.
 LHCb has two RICH detectors: RICH1, located between the [VELO](v.md#VELO) and the [TT](t.md#TT); and RICH2, located after the tracking stations.
+
+## RRB: LHC experiments Resource Review Board {#RRB}
+
+The CERN committee responsible for overseeing the finances of the LHC experiments,
+both for operational costs and for the construction of the detectors and their major upgrades.
+The committee is attended by representatives of all the funding agencies that support the experiment.
```
Comment (alexpearce) [inline glossary/r.md]: Is there one RRB per experiment? Or one for all experiments? If the latter, this should be 'experiment**s**'.

File: glossary/t.md [hunk 5]
```diff
@@ -17,6 +17,30 @@ Advisory body to the LHCb management on all aspects of detector design, optimiza
 
 A unique 32-bit configuration value (in base 16) representing a given trigger configuration (sequence of algorithms and selection cuts).
 
+## TDR: Technical Design Report {#TDR}
+
+A document submitted to the LHCC of CERN describing the design of a subdetector or other subsystem (e.g. online, trigger, computing).
```
Comment (alexpearce) [inline glossary/t.md]: Could link LHCC in this sentence rather than the last one.

File: glossary/t.md [hunk 6]
```diff
@@ -17,6 +17,30 @@ Advisory body to the LHCb management on all aspects of detector design, optimiza
...
+This document also describes the cost of the detector and which institutes will be responsible for which elements of the construction.
+The TDRs of LHCb and the upgrades are collected [here](http://cdsweb.cern.ch/search?cc=LHCb+Reports&ln=en&jrec=11).
+The document is reviewed by the [LHCC](l.md#LHCC), with involvement from the [RRB](r.md#RRB) for financial aspects.
+
+## Technical Proposal {#TP}
+
+A document submitted to the LHCC of CERN describing the design of a new experiment or major upgrade,
```
Comment (alexpearce) [inline glossary/t.md]: Link LHCC.

File: glossary/t.md [hunk 7]
```diff
@@ -17,6 +17,30 @@ Advisory body to the LHCb management on all aspects of detector design, optimiza
...
+outlining the physics case and detector design.
+This document contains more detail than the [LoI](l.md#LoI), for example reporting on R&D
```
Comment (alexpearce) [inline glossary/t.md]: Maybe link to the [Wikipedia article on R&D](https://en.wikipedia.org/wiki/Research_and_development).

File: glossary/l.md (hunk 1 above)
Comment (eduardo-rodrigues) [reply glossary/l.md]: Thanks for the quick look.

With so many cross-links I managed to mis LHCC - fresh eyes are always good!

File: glossary/e.md (hunk 3 above)
Comment (eduardo-rodrigues) [reply glossary/e.md]: Good point.

File: glossary/r.md (hunk 4 above)
Comment (eduardo-rodrigues) [reply glossary/r.md]: It's for all experiments. I will make it clear.

File: glossary/m.md (hunk 2 above)
Comment (eduardo-rodrigues) [reply glossary/m.md]: Fair enough.

Comment (eduardo-rodrigues) [reply glossary/m.md]: Wait, better be a bit vague because small equipment at universities, for example, is part of the institute cost, so not in.

**Comment by alexpearce** [comment]
LGTM, thanks again.

**Pull request merged**